services/harmony/app/util/parameter-parsers.ts

```diff
@@ -79,10 +79,25 @@ export function handleCrs(
  */
 export function handleScaleExtent(
   operation: DataOperation,
-  query: Record<string, string>): void {
+  query: Record<string, number[]>): void;
+export function handleScaleExtent(
+  operation: DataOperation,
+  query: Record<string, string>): void;
+export function handleScaleExtent(
+  operation: DataOperation,
+  query: Record<string, number[] | string>): void {
   if (query.scaleextent) {
-    const [xMin, yMin, xMax, yMax] = query.scaleextent;
-    operation.scaleExtent = { x: { min: xMin, max: xMax }, y: { min: yMin, max: yMax } };
+    if (typeof query.scaleextent === 'string') {
+      const scaleExtentString = query.scaleextent.replace('(', '').replace(')', '');
+      const [xMinString, yMinString, xMaxString, yMaxString] = scaleExtentString.split(/,\s*/);
```

Comment:
The value I found passed around within my local Harmony instance had numbers separated by ", " (so including a single space), but the tests were just comma-delimited. To be safe, I just opted to make the delimiter be a comma with an optional amount of whitespace after it. (Might be a bodge, but works in both my local instance and the tests)